pkgs/tools/audio/alsaequal/default.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv, fetchurl
+, alsaLib, caps
+, alsaUtils
+}:
+
+stdenv.mkDerivation rec {
+  name = "alsaequal";
+  version = "0.6";
+
+  src = fetchurl {
+    url = "https://thedigitalmachine.net/tools/alsaequal-${version}.tar.bz2";
+    sha256 = "1w3g9q5z3nrn3mwdhaq6zsg0jila8d102dgwgrhj9vfx58apsvli";
+  };
+
+  buildInputs = [ alsaLib ];
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  patches = [ ./makefile.patch ./false_error.patch ./caps_9.x.patch ];
+
+  postPatch = ''
+    sed -i 's#/usr/lib/ladspa/caps\.so#${caps}/lib/ladspa/caps\.so#g' ctl_equal.c pcm_equal.c
+  '';
+
+  preInstall = ''
+    mkdir -p "$out/lib/alsa-lib"
+  '';
+
+  postInstall = ''
+    mkdir -p "$out/etc"
```

Comment:
IMO this part seems more appropriate to have in a module.

Reply:
Related to the issue below.